shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1208,7 +1208,23 @@ private boolean performCursorMoveAction(
         arguments.getBoolean(AccessibilityNodeInfo.ACTION_ARGUMENT_EXTEND_SELECTION_BOOLEAN);
     // The voice access expects the semantics node to update immediately. We update the semantics
     // node based on prediction. If the result is incorrect, it will be updated in the next frame.
+    final int previousTextSelectionBase = semanticsNode.textSelectionBase;
+    final int previousTextSelectionExtent = semanticsNode.textSelectionExtent;
```

Comment:
could we add a comment about the rationale for using the previous value instead of the new value?